scripts/scripts.js

```diff
@@ -557,7 +557,7 @@ window.store = new (class {
     this._blockQueryLimit = {
       hero: (block) => {
         const firstHero = document.querySelector('.hero.block[data-block-name="hero"]');
-        if (firstHero.isEqualNode(block)) {
+        if (firstHero?.isEqualNode(block)) {
```

Comment:
This shouldn't be necessary right ? In the sense that hero will only be called if there's a hero on the page so `firstHero` can never be null ?

Reply:
yeah thats true, now that the queryselector is updated. I can change it back